pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -48,25 +46,6 @@ def __init__(self, trainer: "pl.Trainer", log_gpu_memory: Optional[str] = None)
         self._batch_idx: Optional[int] = None
         self._split_idx: Optional[int] = None
 
-    def on_trainer_init(
```

Comment:
In #10417 we propose to make the logger owned by the connector. The two methods `on_trainer_init` and `configure_logger` would then essentially become part of the LoggerConnector init without reference to the trainer. What are your thoughts?

Reply:
I would definitely prefer this approach. Right now, the Trainer is starting to become a bigger class and it is getting harder for users to parse it.